services/121-service/src/scripts/sql/dummy-payment-transactions.sql

```diff
@@ -0,0 +1,33 @@
+        INSERT INTO "121-service"."transaction"
```

Comment:
There are ways to auto-format SQL we could use:

https://www.npmjs.com/package/prettier-plugin-sql
or https://www.npmjs.com/package/prettier-plugin-sql-cst